### ISSUE TITLE
Security integration tests fail in istiodremote topology

### ISSUE THREAD

**Issue by frankbu** [istio/istio#35204]
### Bug Description

The security-istiodremote integration test job is consistently failing.

https://prow.istio.io/?job=integ-security-istiodremote-tests_istio_postsubmit

The following 5 test suites are the cause:
```
istio.io/istio/tests/integration/security/ca_custom_root
istio.io/istio/tests/integration/security/chiron
istio.io/istio/tests/integration/security/filebased_tls_origination
istio.io/istio/tests/integration/security/mtls_first_party_jwt
istio.io/istio/tests/integration/security/sds_ingress
```
These suites are all failing because of what seems to be the same setup problem while installing the gateways:
```
 2021-09-15T11:49:04.751775Z	info	tf	Deploying ingress and egress gateways in config: [-f /home/prow/go/src/istio.io/istio/tests/integration/iop-externalistiod-remote-integration-test-gateways.yaml --istioNamespace istio-system --manifests /home/prow/go/src/istio.io/istio/manifests --set values.global.imagePullPolicy=IfNotPresent]
2021-09-15T11:49:04.849789Z	info	tf	Installing Istio components on cluster config [install --skip-confirmation -f /home/prow/go/src/istio.io/istio/tests/integration/iop-externalistiod-remote-integration-test-gateways.yaml --istioNamespace istio-system --manifests /home/prow/go/src/istio.io/istio/manifests --set values.global.imagePullPolicy=IfNotPresent]
- Processing resources for Egress gateways, Ingress gateways.
- Processing resources for Egress gateways, Ingress gateways. Waiting for Deployment/istio-system...
✘ Egress gateways encountered an error: failed to wait for resource: resources not ready after 5m0s: timed out waiting for the condition
  Deployment/istio-system/istio-egressgateway (containers with unready status: [istio-proxy])
- Processing resources for Ingress gateways. Waiting for Deployment/istio-system/istio-ingressgat...
✘ Ingress gateways encountered an error: failed to wait for resource: resources not ready after 5m0s: timed out waiting for the condition
  Deployment/istio-system/istio-ingressgateway (containers with unready status: [istio-proxy])
```

### Version

```prose
latest
```

**Comment by AndreaM12345** [comment]
running all will fail...but running individually works ... so seems like some earlier test has put it in a bad state...

**Comment by AndreaM12345** [comment]
go test -p 1 -vet=off -v -count=1 -tags=integ ./tests/integration/security/chiron -timeout 30m --istio.test.ci --istio.test.pullpolicy=IfNotPresent --istio.test.kube.topology=/work/localtest.external-istiod.json --istio.test.skipVM
2021-09-23T22:23:05.029102Z	info	tf	Using clusterConfigs file: /work/localtest.external-istiod.json.
2021-09-23T22:23:05.029252Z	info	tf	=== Test Framework Settings ===
2021-09-23T22:23:05.029294Z	info	tf	TestID:            security_chiron
RunID:             7e6f1960-e0fd-49db-9f8c-8ebdaa87af3d
NoCleanup:         false
BaseDir:           /tmp
Selector:          
FailOnDeprecation: false
CIMode:            true
Retries:           0
StableNamespaces:  false
Revision:          
SkipVM:            true
Compatibility:     false
Revisions:         

2021-09-23T22:23:05.029366Z	info	tf	===============================
2021-09-23T22:23:05.035583Z	info	tf	Test run dir: /tmp/security-chiron-7e6f1960e0fd49d
2021-09-23T22:23:05.040257Z	info	tf	Test Framework Kubernetes environment Settings:
Kubeconfigs:           []
LoadBalancerSupported:      true
ControlPlaneTopology: map[]
NetworkTopology:      map[]
ConfigTopology:      map[]

2021-09-23T22:23:05.040340Z	info	tf	=== BEGIN: Building clusters ===
2021-09-23T22:23:05.047776Z	info	tf	Built Cluster:
Name:               config
StableName:         config-0
Kind:               Kubernetes
PrimaryCluster:     external
ConfigCluster:      config
Network:            network-1
HTTPProxy:            
Filename:           /work/artifacts/kubeconfig/config

2021-09-23T22:23:05.047842Z	info	tf	Built Cluster:
Name:               external
StableName:         externalistiod-1
Kind:               Kubernetes
PrimaryCluster:     external
ConfigCluster:      config
Network:            network-1
HTTPProxy:            
Filename:           /work/artifacts/kubeconfig/external

2021-09-23T22:23:05.047861Z	info	tf	=== DONE: Building clusters ===
2021-09-23T22:23:05.047895Z	info	tf	=== BEGIN: Setup: 'security_chiron' ===
2021-09-23T22:23:05.060882Z	info	tf	=== BEGIN: Deploy Istio [Suite=security_chiron] ===
2021-09-23T22:23:05.060939Z	info	tf	=== Istio Component Config ===
2021-09-23T22:23:05.060994Z	info	tf	
SystemNamespace:                istio-system
IstioNamespace:                 istio-system
ConfigNamespace:                istio-system
TelemetryNamespace:             istio-system
PolicyNamespace:                istio-system
IngressNamespace:               istio-system
EgressNamespace:                istio-system
DeployIstio:                    true
DeployEastWestGW:               true
DeployTimeout:                  10m0s
UndeployTimeout:                15m0s
Values:                         map[global.hub:gcr.io/istio-testing global.imagePullPolicy:Always global.tag:latest]
PrimaryClusterIOPFile:          tests/integration/iop-externalistiod-primary-integration-test-defaults.yaml
ConfigClusterIOPFile:           tests/integration/iop-externalistiod-config-integration-test-defaults.yaml
RemoteClusterIOPFile:           tests/integration/iop-externalistiod-remote-integration-test-defaults.yaml
SkipWaitForValidationWebhook:   false
DeployHelm:                     false
DumpKubernetesManifests:        false
IstiodlessRemotes:              false
OperatorOptions:                map[]
EnableCNI:                      false

2021-09-23T22:23:05.061010Z	info	tf	================================
2021-09-23T22:23:05.066291Z	info	proto: tag has too few fields: "-"
2021-09-23T22:23:08.001367Z	info	tf	setting up config as config cluster
2021-09-23T22:23:08.499897Z	info	tf	Installing Istio components on cluster config [install --skip-confirmation -f /work/tests/integration/iop-externalistiod-config-integration-test-defaults.yaml -f /tmp/security-chiron-7e6f1960e0fd49d/_suite_context/istio-deployment-2785570767/iop.yaml --set values.global.imagePullPolicy=IfNotPresent --manifests /work/manifests --set values.global.hub=gcr.io/istio-testing --set values.global.tag=latest --set values.global.imagePullPolicy=Always --set values.global.multiCluster.clusterName=config --set values.istiodRemote.injectionPath=/inject/net/network-1/cluster/config]

- Processing resources for Istiod remote.
✔ Istiod remote installed
- Pruning removed resources
✔ Installation complete2021-09-23T22:23:14.501631Z	error	klog	couldn't get resource list for metrics.k8s.io/v1beta1: the server is currently unable to handle the request
2021-09-23T22:23:14.532333Z	error	klog	couldn't get resource list for metrics.k8s.io/v1beta1: the server is currently unable to handle the request
2021-09-23T22:23:14.571648Z	info	tf	setting up external as control-plane cluster
2021-09-23T22:23:14.573301Z	info	tf	configuring external control plane in external to use config cluster config
2021-09-23T22:23:15.219929Z	info	tf	Installing Istio components on cluster external [install --skip-confirmation -f /work/tests/integration/iop-externalistiod-primary-integration-test-defaults.yaml -f /tmp/security-chiron-7e6f1960e0fd49d/_suite_context/istio-deployment-2785570767/iop.yaml --set values.global.imagePullPolicy=IfNotPresent --manifests /work/manifests --set values.global.imagePullPolicy=Always --set values.global.hub=gcr.io/istio-testing --set values.global.tag=latest --set values.pilot.env.EXTERNAL_ISTIOD=true --set values.global.multiCluster.clusterName=config]

- Processing resources for Istio core.
✔ Istio core installed
- Processing resources for Istiod.
- Processing resources for Istiod. Waiting for Deployment/istio-system/istiod
✔ Istiod installed
- Pruning removed resources
✔ Installation complete2021-09-23T22:23:41.341481Z	info	tf	patching custom host for cluster external as 172.20.255.165
2021-09-23T22:23:46.423245Z	info	tf	creating endpoints and service in config to get discovery from 172.20.255.165
2021-09-23T22:23:46.481843Z	info	tf	Deploying ingress and egress gateways in config: [-f /work/tests/integration/iop-externalistiod-remote-integration-test-gateways.yaml --istioNamespace istio-system --manifests /work/manifests --set values.global.imagePullPolicy=IfNotPresent]
2021-09-23T22:23:46.833574Z	info	tf	Installing Istio components on cluster config [install --skip-confirmation -f /work/tests/integration/iop-externalistiod-remote-integration-test-gateways.yaml --istioNamespace istio-system --manifests /work/manifests --set values.global.imagePullPolicy=IfNotPresent]

- Processing resources for Egress gateways, Ingress gateways.
- Processing resources for Egress gateways, Ingress gateways. Waiting for Deployment/istio-system/istio-egressgateway
- Processing resources for Egress gateways, Ingress gateways. Waiting for Deployment/istio-system/istio-egressgateway, Deployment/istio-system/istio-ingressgateway
✔ Egress gateways installed
✔ Ingress gateways installed
- Pruning removed resources
✔ Installation complete2021-09-23T22:24:07.295799Z	error	klog	couldn't get resource list for metrics.k8s.io/v1beta1: the server is currently unable to handle the request
2021-09-23T22:24:07.302549Z	error	klog	couldn't get resource list for metrics.k8s.io/v1beta1: the server is currently unable to handle the request
2021-09-23T22:24:07.363735Z	info	tf	=== SUCCEEDED: Deploy Istio in 1m2.3723077s [Suite=security_chiron]===
2021-09-23T22:24:07.363969Z	info	tf	=== DONE: Setup: 'security_chiron' (1m2.3854671s) ===
2021-09-23T22:24:07.364201Z	info	tf	=== BEGIN: Test Run: 'security_chiron' ===
=== RUN   TestDNSCertificate
2021-09-23T22:24:07.378749Z	info	tf	=== BEGIN: Test: 'security_chiron[TestDNSCertificate]' ===
=== RUN   TestDNSCertificate/generateDNSCertificates
2021-09-23T22:24:07.388810Z	info	tf	=== BEGIN: Test: 'security_chiron[TestDNSCertificate/generateDNSCertificates]' ===
    dns_cert_test.go:107: check that DNS certificates have been generated ...
    dns_cert_test.go:108: secret dns.istio-galley-service-account did not become existent within 20s
2021-09-23T22:24:27.359868Z	info	tf	=== DONE (failed):  Test: 'security_chiron[TestDNSCertificate/generateDNSCertificates] (20.0057155s)' ===
2021-09-23T22:24:27.360267Z	error	tf	=== Dumping Istio Deployment State...
2021-09-23T22:24:28.813733Z	warn	tf	failed dumping "configz": no running Istio pods in "istio-system"
=== RUN   TestDNSCertificate/regenerateDNSCertificates
2021-09-23T22:24:28.843861Z	info	tf	=== BEGIN: Test: 'security_chiron[TestDNSCertificate/regenerateDNSCertificates]' ===
    dns_cert_test.go:122: sleep 5 seconds for the certificate regeneration to take place ...
    dns_cert_test.go:124: secret dns.istio-galley-service-account did not become existent within 20s
2021-09-23T22:24:53.844445Z	info	tf	=== DONE (failed):  Test: 'security_chiron[TestDNSCertificate/regenerateDNSCertificates] (25.0351487s)' ===
2021-09-23T22:24:53.844933Z	error	tf	=== Dumping Istio Deployment State...
2021-09-23T22:24:55.018002Z	warn	tf	failed dumping "configz": no running Istio pods in "istio-system"
=== RUN   TestDNSCertificate/rotateDNSCertificatesWhenCAUpdated
2021-09-23T22:24:55.046611Z	info	tf	=== BEGIN: Test: 'security_chiron[TestDNSCertificate/rotateDNSCertificatesWhenCAUpdated]' ===
2021-09-23T22:24:55.046773Z	info	tf	=== DONE (passed):  Test: 'security_chiron[TestDNSCertificate/rotateDNSCertificatesWhenCAUpdated] (160.9µs)' ===
--- FAIL: TestDNSCertificate (47.75s)
    --- FAIL: TestDNSCertificate/generateDNSCertificates (21.48s)
    --- FAIL: TestDNSCertificate/regenerateDNSCertificates (26.24s)
    --- FAIL: TestDNSCertificate/rotateDNSCertificatesWhenCAUpdated (0.01s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x120 pc=0x420888d]

goroutine 4278 [running]:
testing.tRunner.func1.2(0x45fa180, 0x6da6870)
	/usr/local/go/src/testing/testing.go:1143 +0x332
testing.tRunner.func1(0xc000dc6000)
	/usr/local/go/src/testing/testing.go:1146 +0x4b6
panic(0x45fa180, 0x6da6870)
	/usr/local/go/src/runtime/panic.go:965 +0x1b9
istio.io/istio/tests/integration/security/chiron_test.TestDNSCertificate.func1.3(0x5307058, 0xc0020b8230)
	/work/tests/integration/security/chiron/dns_cert_test.go:135 +0xcd
istio.io/istio/pkg/test/framework.(*testImpl).doRun(0xc001caccf0, 0xc0020b8230, 0xc001dca280, 0x1953000)
	/work/pkg/test/framework/test.go:319 +0x347
istio.io/istio/pkg/test/framework.(*testImpl).runInternal.func1(0xc000dc6000)
	/work/pkg/test/framework/test.go:223 +0x90
testing.tRunner(0xc000dc6000, 0xc0015cedb0)
	/usr/local/go/src/testing/testing.go:1193 +0xef
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1238 +0x2b3
FAIL	istio.io/istio/tests/integration/security/chiron	110.354s
FAIL
build-tools:/work#

**Comment by AndreaM12345** [comment]
go test -p 1 -vet=off -v -count=1 -tags=integ ./tests/integration/security/filebased_tls_origination -timeout 30m --istio.test.ci --istio.test.pullpolicy=IfNotPresent --istio.test.kube.topology=/work/localtest.external-istiod.json --istio.test.skipVM
2021-09-23T22:32:00.730056Z	info	tf	Using clusterConfigs file: /work/localtest.external-istiod.json.
2021-09-23T22:32:00.730800Z	info	tf	=== Test Framework Settings ===
2021-09-23T22:32:00.731417Z	info	tf	TestID:            security_filebased_tls_origination
RunID:             80de2f34-6272-46b2-8e77-932163c38928
NoCleanup:         false
BaseDir:           /tmp
Selector:          
FailOnDeprecation: false
CIMode:            true
Retries:           0
StableNamespaces:  false
Revision:          
SkipVM:            true
Compatibility:     false
Revisions:         

2021-09-23T22:32:00.731626Z	info	tf	===============================
2021-09-23T22:32:00.734582Z	info	tf	Test run dir: /tmp/security-filebased-tls-origination-
2021-09-23T22:32:00.737138Z	info	tf	Test Framework Kubernetes environment Settings:
Kubeconfigs:           []
LoadBalancerSupported:      true
ControlPlaneTopology: map[]
NetworkTopology:      map[]
ConfigTopology:      map[]

2021-09-23T22:32:00.737382Z	info	tf	=== BEGIN: Building clusters ===
2021-09-23T22:32:00.747695Z	info	tf	Built Cluster:
Name:               config
StableName:         config-0
Kind:               Kubernetes
PrimaryCluster:     external
ConfigCluster:      config
Network:            network-1
HTTPProxy:            
Filename:           /work/artifacts/kubeconfig/config

2021-09-23T22:32:00.748391Z	info	tf	Built Cluster:
Name:               external
StableName:         externalistiod-1
Kind:               Kubernetes
PrimaryCluster:     external
ConfigCluster:      config
Network:            network-1
HTTPProxy:            
Filename:           /work/artifacts/kubeconfig/external

2021-09-23T22:32:00.748684Z	info	tf	=== DONE: Building clusters ===
2021-09-23T22:32:00.749414Z	info	tf	=== BEGIN: Setup: 'security_filebased_tls_origination' ===
2021-09-23T22:32:00.856287Z	info	tf	=== SUCCESS: context setup function ===
2021-09-23T22:32:00.856374Z	info	tf	=== BEGIN: Deploy Istio [Suite=security_filebased_tls_origination] ===
2021-09-23T22:32:00.856404Z	info	tf	=== Istio Component Config ===
2021-09-23T22:32:00.856480Z	info	tf	
SystemNamespace:                istio-system
IstioNamespace:                 istio-system
ConfigNamespace:                istio-system
TelemetryNamespace:             istio-system
PolicyNamespace:                istio-system
IngressNamespace:               istio-system
EgressNamespace:                istio-system
DeployIstio:                    true
DeployEastWestGW:               true
DeployTimeout:                  10m0s
UndeployTimeout:                15m0s
Values:                         map[global.hub:gcr.io/istio-testing global.imagePullPolicy:Always global.tag:latest]
PrimaryClusterIOPFile:          tests/integration/iop-externalistiod-primary-integration-test-defaults.yaml
ConfigClusterIOPFile:           tests/integration/iop-externalistiod-config-integration-test-defaults.yaml
RemoteClusterIOPFile:           tests/integration/iop-externalistiod-remote-integration-test-defaults.yaml
SkipWaitForValidationWebhook:   false
DeployHelm:                     false
DumpKubernetesManifests:        false
IstiodlessRemotes:              false
OperatorOptions:                map[]
EnableCNI:                      false

2021-09-23T22:32:00.856499Z	info	tf	================================
2021-09-23T22:32:00.864662Z	info	proto: tag has too few fields: "-"
2021-09-23T22:32:03.902610Z	info	tf	setting up config as config cluster
2021-09-23T22:32:04.428493Z	info	tf	Installing Istio components on cluster config [install --skip-confirmation -f /work/tests/integration/iop-externalistiod-config-integration-test-defaults.yaml -f /tmp/security-filebased-tls-origination-/_suite_context/istio-deployment-3306640311/iop.yaml --set values.global.imagePullPolicy=IfNotPresent --manifests /work/manifests --set values.global.hub=gcr.io/istio-testing --set values.global.tag=latest --set values.global.imagePullPolicy=Always --set values.global.multiCluster.clusterName=config --set values.istiodRemote.injectionPath=/inject/net/network-1/cluster/config]

- Processing resources for Egress gateways.
- Processing resources for Egress gateways, Istiod remote.
- Processing resources for Egress gateways, Istiod remote. Waiting for Deployment/istio-system/istio-egressgateway
✔ Istiod remote installed
- Processing resources for Egress gateways. Waiting for Deployment/istio-system/istio-egressgateway
✘ Egress gateways encountered an error: failed to wait for resource: resources not ready after 5m0s: timed out waiting for the condition
  Deployment/istio-system/istio-egressgateway (container failed to start: ContainerCreating: )
- Pruning removed resources2021-09-23T22:37:06.952695Z	info	tf	=== FAILED: Deploy Istio in 5m6.4430456s [Suite=security_filebased_tls_origination] ===
2021-09-23T22:37:06.952843Z	error	tf	=== Dumping Istio Deployment State...
2021-09-23T22:37:07.086790Z	warn	tf	Unable to get logs for pod/container: istio-system/istio-egressgateway-85d9cdf5c8-w4965/istio-proxy for: container "istio-proxy" in pod "istio-egressgateway-85d9cdf5c8-w4965" is waiting to start: ContainerCreating
2021-09-23T22:37:07.104632Z	warn	tf	Unable to get core dumps for pod: istio-system/istio-egressgateway-85d9cdf5c8-w4965: error exec'ing into istio-system/istio-egressgateway-85d9cdf5c8-w4965 istio-proxy container: unable to upgrade connection: container not found ("istio-proxy")
2021-09-23T22:37:07.107790Z	error	tf	Unable to get execute command "pilot-agent request --debug-port 15020 GET /debug/ndsz" on pod: istio-system/istio-egressgateway-85d9cdf5c8-w4965 for: error exec'ing into istio-system/istio-egressgateway-85d9cdf5c8-w4965 istio-proxy container: unable to upgrade connection: container not found ("istio-proxy")
2021-09-23T22:37:07.108901Z	error	tf	Unable to get execute command "pilot-agent request GET config_dump?include_eds=true" on pod: istio-system/istio-egressgateway-85d9cdf5c8-w4965 for: error exec'ing into istio-system/istio-egressgateway-85d9cdf5c8-w4965 istio-proxy container: unable to upgrade connection: container not found ("istio-proxy")
2021-09-23T22:37:07.138384Z	error	tf	Unable to get execute command "pilot-agent request GET clusters" on pod: istio-system/istio-egressgateway-85d9cdf5c8-w4965 for: error exec'ing into istio-system/istio-egressgateway-85d9cdf5c8-w4965 istio-proxy container: unable to upgrade connection: container not found ("istio-proxy")
2021-09-23T22:37:07.188806Z	warn	tf	failed dumping "configz": no running Istio pods in "istio-system"
2021-09-23T22:37:07.203778Z	error	tf	Test setup error: install failed: failed to install manifests: errors occurred during operation
2021-09-23T22:37:07.203836Z	info	tf	=== FAILED: Setup: 'security_filebased_tls_origination' (install failed: failed to install manifests: errors occurred during operation) ===
2021-09-23T22:37:07.203859Z	error	tf	Exiting due to setup failure: install failed: failed to install manifests: errors occurred during operation
2021-09-23T22:37:07.203908Z	error	tf	=== Dumping Istio Deployment State...
2021-09-23T22:37:07.251572Z	warn	tf	Unable to get logs for pod/container: istio-system/istio-egressgateway-85d9cdf5c8-w4965/istio-proxy for: container "istio-proxy" in pod "istio-egressgateway-85d9cdf5c8-w4965" is waiting to start: ContainerCreating
2021-09-23T22:37:07.275364Z	error	tf	Unable to get execute command "pilot-agent request --debug-port 15020 GET /debug/ndsz" on pod: istio-system/istio-egressgateway-85d9cdf5c8-w4965 for: error exec'ing into istio-system/istio-egressgateway-85d9cdf5c8-w4965 istio-proxy container: unable to upgrade connection: container not found ("istio-proxy")
2021-09-23T22:37:07.285380Z	warn	tf	Unable to get core dumps for pod: istio-system/istio-egressgateway-85d9cdf5c8-w4965: error exec'ing into istio-system/istio-egressgateway-85d9cdf5c8-w4965 istio-proxy container: unable to upgrade connection: container not found ("istio-proxy")
2021-09-23T22:37:07.289129Z	error	tf	Unable to get execute command "pilot-agent request GET config_dump?include_eds=true" on pod: istio-system/istio-egressgateway-85d9cdf5c8-w4965 for: error exec'ing into istio-system/istio-egressgateway-85d9cdf5c8-w4965 istio-proxy container: unable to upgrade connection: container not found ("istio-proxy")
2021-09-23T22:37:07.325952Z	error	tf	Unable to get execute command "pilot-agent request GET clusters" on pod: istio-system/istio-egressgateway-85d9cdf5c8-w4965 for: error exec'ing into istio-system/istio-egressgateway-85d9cdf5c8-w4965 istio-proxy container: unable to upgrade connection: container not found ("istio-proxy")
2021-09-23T22:37:07.395805Z	warn	tf	failed dumping "configz": no running Istio pods in "istio-system"
2021-09-23T22:37:07.407666Z	info	tf	=== BEGIN: Cleanup Istio [Suite=security_filebased_tls_origination] ===
2021-09-23T22:37:07.477959Z	error	klog	couldn't get resource list for metrics.k8s.io/v1beta1: the server is currently unable to handle the request
2021-09-23T22:37:07.483078Z	error	klog	couldn't get resource list for metrics.k8s.io/v1beta1: the server is currently unable to handle the request
2021-09-23T22:37:07.943973Z	info	tf	=== SUCCEEDED: Cleanup Istio in 529.1566ms [Suite=security_filebased_tls_origination] ===
FAIL	istio.io/istio/tests/integration/security/filebased_tls_origination	307.840s
FAIL

**Comment by AndreaM12345** [comment]
go test -p 1 -vet=off -v -count=1 -tags=integ ./tests/integration/security/sds_ingress -timeout 30m --istio.test.ci --istio.test.pullpolicy=IfNotPresent --istio.test.kube.topology=/work/localtest.external-istiod.json --istio.test.skipVM
2021-09-23T22:48:10.964263Z	info	tf	Using clusterConfigs file: /work/localtest.external-istiod.json.
2021-09-23T22:48:10.967372Z	info	tf	=== Test Framework Settings ===
2021-09-23T22:48:10.969485Z	info	tf	TestID:            security_sds_ingress
RunID:             ca6510e4-38d5-42c3-8092-026361f04d61
NoCleanup:         false
BaseDir:           /tmp
Selector:          
FailOnDeprecation: false
CIMode:            true
Retries:           0
StableNamespaces:  false
Revision:          
SkipVM:            true
Compatibility:     false
Revisions:         

2021-09-23T22:48:10.972839Z	info	tf	===============================
2021-09-23T22:48:10.983047Z	info	tf	Test run dir: /tmp/security-sds-ingress-ca6510e438
2021-09-23T22:48:11.002140Z	info	tf	Test Framework Kubernetes environment Settings:
Kubeconfigs:           []
LoadBalancerSupported:      true
ControlPlaneTopology: map[]
NetworkTopology:      map[]
ConfigTopology:      map[]

2021-09-23T22:48:11.005477Z	info	tf	=== BEGIN: Building clusters ===
2021-09-23T22:48:11.012088Z	info	tf	=== FAILED: Building clusters ===
2021-09-23T22:48:11.014161Z	error	tf	Error during test framework init: 2 errors occurred:
	* failed building cluster from config 0: failed normalizing file /work/artifacts/kubeconfig/config: stat /work/artifacts/kubeconfig/config: no such file or directory
	* failed building cluster from config 1: failed normalizing file /work/artifacts/kubeconfig/external: stat /work/artifacts/kubeconfig/external: no such file or directory


FAIL	istio.io/istio/tests/integration/security/sds_ingress	0.449s
FAIL

**Comment by frankbu** [comment]
This is mostly fixed by several other PRs, and followup in #36002.
